microsoft-power-point/microsoft-power-point-quiz.md

```diff
@@ -1,6 +1,6 @@
 ## Microsoft PowerPoint
 
-#### Q1. When you create a new, blank presentation, you begin with one blank slide. What is the layout name of this slide?
+(https://www.bzziii.com/2022/08/when-you-create-new-blank-presentation-you-begin-with-one-blank-slide-what-is-the-layout-name-of-this-slide.html) Q1. When you create a new, blank presentation, you begin with one blank slide. What is the layout name of this slide?
```

Comment:
Hello, @dipankar075! Thanks for reference link, could you please move it from the beginning to the end of this question Q1, please check some other similar examples. Will wait for your update, thanks